pages/apim/3.x/kubernetes/apim-kubernetes-operator-installation.adoc

```diff
@@ -0,0 +1,41 @@
+[[apim-kubernetes-operator-installation]]
+= GKO installation and deployment
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_kubernetes_operator_installation.html
+:page-folder: apim/kubernetes
+:page-layout: apim3x
+
+You can deploy the Gravitee Kubernetes Operator [GKO] in two ways:
+
+1. On a production Kubernetes cluster.
```

Comment:
```suggestion
1. On a remote Kubernetes cluster.
```
I would talk more about remote and local. And reserve production to subject of performance or security or other. to avoid some confusions

Reply:
Good point - updated where needed